mesonbuild/compilers/mixins/visualstudio.py

```diff
@@ -137,6 +138,11 @@ def __init__(self, target: str):
 
     # Override CCompiler.get_always_args
     def get_always_args(self) -> T.List[str]:
+        if not isinstance(self, ClangClCompiler) and mesonlib.version_compare(self.version, '<19.00'):
+            try:
+                self.always_args.remove('/utf-8')
+            except ValueError:
+                pass
```

Comment:
Exceptions are slow, you can instead do:

```py
if '/utf-8' in self.always_args:
    self.always_args.remove('/utf-8')
```

Reply:
Thanks for the tip, @nirbheek!